### ISSUE TITLE
The GITEA_RUNNER_GID envar, another attempt.

### ISSUE THREAD

**Issue by wblew** [vegardit/docker-gitea-act-runner#9]
Follow-up to closed Issue https://github.com/vegardit/docker-gitea-act-runner/issues/5

That issue's PR #6 is broken. The resulting image will not start when the GITEA_RUNNER_GID is present and not 1000.

That is because my testing was inadequate, related to being unable to build the vegardit/gitea-act-runner image locally.

Method of testing: I used the attached the attached docker-compose.yml and Dockerfile.test in the root of this repo.

[docker-compose.yml.txt](https://github.com/vegardit/docker-gitea-act-runner/files/11521626/docker-compose.yml.txt)

[Dockerfile.test.txt](https://github.com/vegardit/docker-gitea-act-runner/files/11521627/Dockerfile.test.txt)

It does work, despite the container having to restart, due to sudo failure at https://github.com/vegardit/docker-gitea-act-runner/blob/4c9806d71c4f319d74b71d63597f0c222e95bda5/image/run.sh#L48

Basically: changing the running UID's /etc/passwd entry causes its next sudo command to fail.

The container restart (without wiping the read/write layer) starts with the already changed /etc/passwd, and then continues without the sudo error.

**Comment by wblew** [comment]
Created [PR #](https://github.com/vegardit/docker-gitea-act-runner/pull/10) that fixes this issue.
